components/navigation/Navigation.html

```diff
@@ -0,0 +1,231 @@
+                <span class="sidebar-item-text">Input</span>
+              </li>
+            </a>
+            <a href="../textUtilities/TextUtilities.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"><i class="fas fa-heading"></i></span>
+                <span class="sidebar-item-text">Text Utilities</span>
+              </li>
+            </a>
+            <a href="../lists/Lists.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"><i class="fas fa-list-ul"></i></span>
+                <span class="sidebar-item-text">Lists</span>
+              </li>
+            </a>
+            <a href="../modal/Modal.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"
+                  ><i class="far fa-credit-card"></i
+                ></span>
+                <span class="sidebar-item-text">Modal</span>
+              </li>
+            </a>
+            <a href="../navigation/Navigation.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"
+                  ><i class="fas fa-directions"></i
+                ></span>
+                <span class="sidebar-item-text">Navigation</span>
+              </li>
+            </a>
+            <a href="../rating/Rating.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"><i class="fas fa-star"></i></span>
+                <span class="sidebar-item-text">Rating</span>
+              </li>
+            </a>
+            <a href="../snackbar/Snackbar.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"
+                  ><i class="fas fa-bread-slice"></i
+                ></span>
+                <span class="sidebar-item-text">Snackbar / Toast</span>
+              </li>
+            </a>
+            <a href="../simplefiedGrid/SimplifiedGrid.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"
+                  ><i class="far fa-credit-card"></i
+                ></span>
+                <span class="sidebar-item-text">Simplified Grid</span>
+              </li>
+            </a>
+          </ul>
+        </nav>
+
+        <!-- image section -->
+        <div class="example-container">
+          <h1 class="example-title">Navbar</h1>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Corporis
+            voluptate iusto at esse voluptatibus itaque molestias veritatis
+            deserunt, blanditiis eaque.
+          </p>
+          <hr class="example-hr" />
+          <h2>Simple Navigation</h2>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Adipisci,
+            nobis.
+          </p>
+
+          <div class="utilities-box">
+            <div class="navigation">
```

Comment:
You can directly use `<nav>` tag here if you want.

Reply:
Yes, thought of that, will make the change.